bigbluebutton-html5/imports/ui/components/common/error-boundary/error-boundary-with-reload/component.jsx

```diff
@@ -60,19 +64,24 @@ const ErrorBoundaryWithReload = ({ children }) => {
           errorStack: event.error?.stack,
         },
       }, 'Global error caught by ErrorBoundaryWithReload');
-    
+
       triggerError();
     };
 
     const handleUnhandledRejection = (event) => {
+      // Ignore error caused by ReactDevTools (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/installHook.js)
+      if (event.reason?.stack?.toString().indexOf('fmkadmapgofadopljbjfkapdkoienihi') !== -1) {
```

Comment:
Could we check `indexOf('chrome-extension') instead to be a bit more generic? I can't think of an extension that we'd want this to be valid for...

Reply:
I agree